tools/image_processing/woundhealing/woundhealing.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="woundhealing" name="Wound healing scratch assay image analysis" version="@VERSION@+galaxy1">
```

Comment:
should we make this tool id a bit more specific? `fiji_woundhealing` or `img_wouldhealing`?